include/tcp_handler.h

```diff
@@ -31,9 +35,13 @@ struct tcp_network_data
     int sock;
 };
 
-void tcp_network_manager(struct tcp_network_data* nm);
+bool tcp_network_manager(struct tcp_network_data* nm);
```

Comment:
Can you use int rather than bool ?

Reply:
there is no point of using int since error status is saved in errno variable by esp. This is just to say if it was created successfully or not. if not, user can see errno for better handling for their project.

Reply:
Since, we need to include <stdbool.h> to use bool in C. I suggested this

Reply:
For ESP you do not need to add any library. I tried this with only C code project and it works fine. Library internally handles it according to C compiler or C++ with same #ifdef __cplusplus check. it works for both of my C and C++ projects.

Reply:
Ohk, then